h/templates/client/selection_tabs.html

```diff
@@ -2,10 +2,26 @@
 <ul class="selection-tabs">
   <li class="selection-tabs__type" ng-class="{'selection-tabs--selected': vm.selectedTab === vm.tabAnnotations}" ng-click="vm.selectTab('annotation')">
     Annotations
-    <sup ng-if="vm.totalAnnotations > 0">{{ vm.totalAnnotations }}</sup>
+    <span class="selection-tabs__count" ng-if="vm.totalAnnotations > 0">{{ vm.totalAnnotations }}</sup>
   </li>
   <li class="selection-tabs__type" ng-class="{'selection-tabs--selected': vm.selectedTab === vm.tabNotes}" ng-click="vm.selectTab('note')">
     Notes
-    <sup ng-if="vm.totalNotes > 0">{{ vm.totalNotes }}</sup>
+    <span class="selection-tabs__count" ng-if="vm.totalNotes > 0">{{ vm.totalNotes }}</sup>
+  </li>
+</ul>
+<ul ng-if="!vm.isLoading()">
```

Comment:
This should probably be just a `<div>` rather than a list. We should only use list elements for things that are actually lists from the user's point of view, which I would hope is more likely to result in helpful behavior from accessibility tools.